controllers/channel.js

```diff
@@ -62,6 +62,9 @@ exports.listChannels = (req,res) => {
                 our_channel_reserve_satoshis: peer.channels[0].our_channel_reserve_satoshis,
                 spendable_msatoshi: peer.channels[0].spendable_msatoshi
             };
+            if (peer.channels[0].direction) {
```

Comment:
Cool, thanks for fixing it. My logic wasn't correct anyway, which I realized after reading your reply to the issue. I assumed `1` meant `true` and `0` meant `false` but it appears to be more complicated than that. Passing along the flag like you did give me more flexibility to test and figure out what the values mean.

My only mild concern about  the line `if (peer.channels[0].direction) {` is whether or not `undefined` has a different meaning than `0` in c-lightning. If so, then I would not be able to make that distinction in my app. The API will not return a value in either case. It may have been better to use `peer.channels[0].direction !== undefined` so that the `0` value would be returned in the response

I'll keep testing and let you know if anything may need to be changed. Thanks for the help